virttest/qemu_virtio_port.py

```diff
@@ -789,12 +789,12 @@ def run_normal(self):
                     # Compare the received data with the control data
                     for char in buf:
                         _char = self.buff.popleft()
-                        if char == _char:
+                        if char == _char or char == ord(_char):
```

Comment:
Using `bytearray` and `struct.pack()` would make an unified approach to process the check.

For example:
```python
for c_code in bytearray(buf):
    char = struct.pack('B', c_code)
    _char = self.buff.popleft()
    if char == _char:
        ...
```

Reply:
@luckyh Great, it works always. I will update it this way. Thanks.